clang/lib/StaticAnalyzer/Checkers/CStringChecker.cpp

```diff
@@ -2463,19 +2473,19 @@ void CStringChecker::evalStrsep(CheckerContext &C,
     // character to NUL.
     // As the replacement never overflows, do not invalidate its super region.
     State = invalidateDestinationBufferNeverOverflows(
-        C, State, SearchStrPtr.Expression, Result);
+        C, State, Call.getCFGElementRef(), Result);
 
     // Overwrite the search string pointer. The new value is either an address
     // further along in the same string, or NULL if there are no more tokens.
     State =
         State->bindLoc(*SearchStrLoc,
-                       SVB.conjureSymbolVal(getTag(), Call.getOriginExpr(),
+                       SVB.conjureSymbolVal(getTag(), Call.getCFGElementRef(),
                                             LCtx, CharPtrTy, C.blockCount()),
                        LCtx);
   } else {
     assert(SearchStrVal.isUnknown());
     // Conjure a symbolic value. It's the best we can do.
-    Result = SVB.conjureSymbolVal(nullptr, Call.getOriginExpr(), LCtx,
+    Result = SVB.conjureSymbolVal(nullptr, Call.getCFGElementRef(), LCtx,
                                   C.blockCount());
   }
```

Comment:
I think it's such a common pattern that we want to conjure at a call, that we should have a shorthand for it.
Conjure takes a bunch of obscured parameters and it's annoying to dig them up every time.
The "Call" should already have most stuff, thus we could have the opportunity to simplify the API for Calls.
It could be done later though. I just leave this for food for though.